pkg/mms/events.go

```diff
@@ -96,6 +114,34 @@ func ListProductEvents(eventCache string, opts Options) ([]*ProductEvent, error)
 	return events, nil
 }
 
+// MakeProductEvent prepares and sends the product event
+func MakeProductEvent(hubs []ProductionHub, p *ProductEvent) error {
+
```

Comment:
Usually Go functions do not have an empty line between function signature and the function body.